app/components/action_button.html.erb

```diff
@@ -1 +1 @@
-<%= link_to label, url, class: "btn button btn-primary #{'disabled' if disabled?}", data: button_data %>
+<%= link_to label, url, method: "#{method}", class: "btn button btn-primary #{'disabled' if disabled?}", data: button_data %>
```

Comment:
I think you can drop the interpolation here? `method: method` is easier, right?

Reply:
yes, good call